Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine3.14
 
 LABEL org.opencontainers.image.source="https://github.com/Countingup/docker-nswag"
 
-RUN apk add --no-cache --update unzip curl busybox \
+RUN apk add --no-cache --update --upgrade ncurses-terminfo-base libssl1.1 libcrypto1.1 unzip curl busybox \
```

Comment:
update & upgrade - does this change what we're doing with the other dependencies? @kreativka perhaps you can chip in here

Reply:
`apk`: `--update` - update list of packages, `--upgrade` - upgrade installed packages. We need to add `--upgrade` here to install new versions.

Reply:
should I add another layer just to upgrade the added packages?